en/technical-reference/sandbox-test.md

```diff
@@ -94,6 +94,12 @@ This is useful for Installment/Promo scenario which require bank specific card.
 |**BRI** | 4365 0286 6251 2583 | 5520 0219 0920 3008|
 |**MEGA** | 4201 9100 0000 0025 | 5221 0300 0000 0025|
 |**Maybank** | 4055 7796 2846 0474 | 5520 0883 1465 3770|
+
+##### Offline Card 
```

Comment:
nice, thanks. please add note so user know the card will not be accepted
```
##### Offline Card 
For testing scenario: Deny because card is not eligible for online transaction
...
```

Reply:
Sure, done ya